packages/components/src/molecules/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,130 @@
+import React, {
+  type ReactNode,
+  type MouseEventHandler,
+  useState,
+  forwardRef,
+  type HTMLAttributes,
+} from 'react'
+import Icon from '../../atoms/Icon'
+import IconButton from '../IconButton'
+/**
+ * Specifies tooltip position.
+ */
+export type Side = 'top' | 'right' | 'bottom' | 'left'
+
+/**
+ * Specifies tooltip alignment.
+ */
+export type Alignment = 'start' | 'end'
+
+/**
+ * Example: "top", "top-start", "top-end", etc.
+ */
+export type AlignedPlacement = `${Side}-${Alignment}`
+
+/**
+ * Combines pure side (e.g., "top") or side + alignment (e.g., "top-start").
+ */
+export type Placement = Side | AlignedPlacement
+
+export interface TooltipProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, 'content'> {
+  /**
+   * Text/content of the tooltip.
+   */
+  content: ReactNode
+  /**
+   * Defines the side or side-alignment (e.g., "top", "right-end") of the tooltip.
+   */
+  placement?: Placement
+  /**
+   * If the tooltip can be closed by a button.
+   */
+  dismissible?: boolean
```

Comment:
@renatamottam do we really need to have this `dismissible` prop?

The tooltip is expected to disappear when the focus is lost or on mouse out. 
And according to this [accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role) doc, it shouldn't contain interactive elements 🤔 So adding a button would be a problem (will add focus to the button). WDYT?

Ps: Add a big notice to the doc: this component should only be used for additional information and not for crucial content, as tooltip content may not be accessible to screen readers and could go unnoticed.